examples/paragraph/frontend/Cargo.toml

```diff
@@ -17,6 +17,7 @@ wasm-bindgen-test = "0.3.19"
 
 [dependencies]
 zoon = { path = "../../../crates/zoon" }
+lang = { path = "../../../crates/lang"}
```

Comment:
~Please move it to Zoon's dependencies. We can make it optional later to make sure long `Lang` enum won't slow down the compiler.~ Then remove the line.